Unix/installbuilder/datafiles/Linux.data

```diff
@@ -14,6 +14,7 @@ SERVICE_CTL:  '/opt/omi/bin/service_control'
 /opt/omi/bin/support/installssllinks;         ../scripts/installssllinks;            755; root; ${{ROOT_GROUP_NAME}}
 /opt/omi/bin/support/omid.systemd;            ../installbuilder/service_scripts/omid.systemd; 644; root; ${{ROOT_GROUP_NAME}}
 /opt/omi/bin/support/omid;                    ../installbuilder/service_scripts/omid.ulinux; 755; root; ${{ROOT_GROUP_NAME}}
```

Comment:
Given that we now have two different startup scripts (three really), I think that this file should be something like `omid.initd` or something so it's clear what it's for ...